integrations/tensorflow/test/CMakeLists.txt

```diff
@@ -5,3 +5,4 @@
 # SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
 
 add_subdirectory(tf_integration)
+add_subdirectory(tflite_integration)
```

Comment:
This directory is half set up for testing... so be careful and check that your new tests are actually running (they aren't right now AFAICT):

https://github.com/google/iree/blob/ebf3a6a25a5a86c5a68db0ee325d741c8bff4d26/integrations/tensorflow/CMakeLists.txt#L20-L28

https://github.com/google/iree/blob/ebf3a6a25a5a86c5a68db0ee325d741c8bff4d26/CMakeLists.txt#L76

also check that test scripts aren't excluding the directory, e.g. audit the filtering here: https://github.com/google/iree/blob/ebf3a6a25a5a86c5a68db0ee325d741c8bff4d26/build_tools/kokoro/gcp_ubuntu/cmake-bazel/linux/x86-swiftshader/build.sh#L72-L75

Reply:
Updated flags and process. Checked and it should run well. None of the tests take that long (minus fetching the files). Tests are verified to pass.

Reply:
yep, new tests are running now: https://source.cloud.google.com/results/invocations/8799ea6c-b50f-4aee-ba88-cd3f4f3c2f2a/targets/iree%2Fgcp_ubuntu%2Fcmake-bazel%2Flinux%2Fx86-swiftshader%2Fpresubmit/log 
```
        Start  930: integrations/tensorflow/test/tflite_integration/mobilenet_v1
        Start  931: integrations/tensorflow/test/tflite_integration/posenet_i8
...
integrations/tensorflow/test/tflite_integration            = 108.59 sec*proc (2 tests)
```